rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/event/RMNodeSourceEvent.java

```diff
@@ -59,6 +59,8 @@
 
     private String nodeSourceAdmin = null;
 
+    private Type type = Type.ADDED;
```

Comment:
Why don't you rely on existing RMEventType? There will be new node source state soon.